GLOSSARY.md

```diff
@@ -2,48 +2,38 @@
-    In these cases, other modes of operations should be considered (e.g. manual intervention), followed by any necessary updates to the desired state declarations, and finally resuming reconciliation of the system with the updated declarations.
-    Pragmatic exceptions to these guiding principles are expected from time to time during the journey toward a system being fully managed by GitOps.
-
 - ## Continuous
 
-    By "continuous" we adopt the industry standard to mean that [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
+    "Continuous" is intended to match the industry standard term: [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
 
 - ## Declarative Description
 
-    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating configuration (the desired state) from the implementation (commands, API calls, scripts etc.) that actually achieves the desired state described in the declarative description.
+    A configuration that describes the desired operating state of a system without specifying procedures for how that state will be achieved. This separates configuration (the desired state) from the implementation (commands, API calls, scripts etc.) used to achieve that state.
 
 - ## Desired State
 
-    The aggregate of all configuration data for a system form its desired state which is defined as data sufficient to recreate the system so that instances of the system are behaviourally indistinguishable, but do not include the state of any data stored within the system, eg. database contents.
+    The aggregate of all configuration data needed to start, bootstrap, recreate, run, and recover a system. This does not include data stored within a system, eg. database contents.
 
 - ## Drift
 
-    When a system's actual state changes for any reason other than its versioned [desired state](#desired-state) declarations having changed, we say that the system has drifted from its desired state.
+    When a system's actual state changes for any reason other than its [desired state](#desired-state)declarations having changed. When this occurs, the system has drifted from its desired state.
 
 - ## Reconciliation
 
-    The process of ensuring that the actual state of a system matches its [desired state](#desired-state) declarations.
-    Contrary to CIops, any divergence between the two will trigger reconciliation, regardless of where changes occured.
-    Divergence could be due to the actual state unintentionally [drifting](#drift) from the desired state declarations, or a new desired state declaration version having been changed intentionally.
+    The process of ensuring the actual state of a system matches its [desired state](#desired-state) declarations.
```

Comment:
(nit?): The phrase `desired state declarations` is used in a couple places in this PR. For some reason it doesn't sit right with me - in particular the word `declarations`. I guess I feel that this is redundant (a form of tautology) in that `desired state` is already defined in this document as:

> The aggregate of all configuration data

Given the plural nature of the word `declarations` does that now mean that we are referring to a collection of `aggregates`?

Reply:
@onematchfox I see your point because `"desired state"` is already defined as a phrase — To a casual reader _skimming_, another way to interpret it is `desired "state declarations"`, which gives it a different meaning.  Perhaps the actual intent is `declared "desired state"`?

Reply:
:thinking: Yeah, I think you've got the basis of the point I'm trying to get across. Although, I would even take it a step further and aim to drop the word "declarative/declared" entirely. In this particular sentence I think it would (should?) be sufficient with saying:

> The process of ensuring that the actual state of a system matches **the** [desired state](#desired-state).

But to achieve this it would probably mean adjusting the definition of "desired state" to make the _declarative_ nature more apparent? In some ways the concept is already there but does require some reading _between the lines_: 

> Declarative Description
> A **configuration** that describes the desired operating state of a system ...
> 
> Desired State
> The aggregate of all **configuration** data

Reply:
@onematchfox LGTM

Reply:
@onematchfox LGTM; would you kindly create an in-line `suggestion` via the `±` icon here in GitHub?  Thank you!

Reply:
Totally agree, merging.

Reply:
@lloydchang we're doing a commit with co-author now.

Reply:
Merged.